assets/app/views/_triggers.html

```diff
@@ -32,14 +32,14 @@
         <span ng-switch-when="Cancelled">was cancelled.</span>
         <span ng-switch-default>is {{build.status.phase | lowercase}}.
           <span ng-if="trigger.imageChangeParams.automatic">
-            A new deployment will be created automatically once the build completes.&nbsp;&nbsp;
-            <a
-              href="/project/{{build.metadata.namespace}}/browse/builds/{{build.metadata.labels.buildconfig}}/{{build.metadata.name}}?tab=logs">
-              View Log
-            </a>
+            A new deployment will be created automatically once the build completes.
           </span>
         </span>
       </span>
+      <a
+        ng-href="project/{{build.metadata.namespace}}/browse/builds/{{build.metadata.labels.buildconfig}}/{{build.metadata.name}}?tab=logs">
+        View Log
+      </a>
```

Comment:
We don't want the link to show if the build phase is `New` or `Pending`. Neither of these had a pod yet, so there's no log.

```
	// BuildPhaseNew is automatically assigned to a newly created build.
	BuildPhaseNew BuildPhase = "New"

	// BuildPhasePending indicates that a pod name has been assigned and a build is
	// about to start running.
	BuildPhasePending BuildPhase = "Pending"
```